pkg/universal_sigver/client.go

```diff
@@ -0,0 +1,85 @@
+package universal_sigver
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/layer-3/clearsync/pkg/smart_wallet"
+	"github.com/shopspring/decimal"
+)
+
+var entryPointV0_6Address = common.HexToAddress("0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789")
+
+type Client interface {
+	Verify(ctx context.Context, signer common.Address, messageHash common.Hash, signature []byte) (bool, error)
+	PackERC6492Sig(ctx context.Context, ownerAddress common.Address, index decimal.Decimal, sig []byte) ([]byte, error)
+}
```

Comment:
It would be beneficial to include a godoc comment here to elucidate the interface's and functions' purpose and context, which might not be immediately obvious considering the topic of the package.

Reply:
6982100528d7e2cfb6f19798fbceeaf3e24497b1